### PR TITLE
Add ehen->when

### PR DESCRIPTION
**Pull request by luzpaz**

### CODE REVIEW

File: codespell_lib/data/dictionary.txt [hunk 1]
```diff
@@ -10188,6 +10188,7 @@ ehance->enhance
 ehanced->enhanced
 ehancement->enhancement
 ehancements->enhancements
+ehen->when
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt:10191]: Suggest hen too as a typo.

Add ehenever as another typo too perhaps?

**Review by peternewman — CHANGES_REQUESTED** [review]
One comment

File: codespell_lib/data/dictionary.txt (hunk 1 above)
Comment (lurch) [reply codespell_lib/data/dictionary.txt:10191]: Could also suggest ehen->even ?

Comment (luzpaz) [reply codespell_lib/data/dictionary.txt]: Added in bb06c83

**Comment by lurch** [comment]
@peternewman Hmm, should `ehen->then` also be suggested, or is that "too distant"? :shrug:

**Comment by peternewman** [comment]
> ehen

Yeah seems like it could possibly happen on a Qwerty keyboard.

**Comment by luzpaz** [comment]
Added in 355a784

**Review by peternewman — APPROVED** [review]

**Pull request merged**